File: Assignment_6/Question3.c
#include<stdio.h>
#include<stdbool.h>

bool CheckEqualnum(int iNo1, int iNo2)
{
    if(iNo1 == iNo2)
    {
        return true;
    }
    else
    {
        return false;
    }
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;
    
    bool iResult = false;

    printf("Enter the first number : \n");
    scanf("%d", &iValue1);

    printf("Enter the second number : \n");
    scanf("%d", &iValue2);

    iResult = CheckEqualnum(iValue1, iValue2);

    if(iResult == true)
    {
        printf("The numbers are equal.");
    }
    else
    {
        printf("The numbers are not equal.");
    }
    return 0;
}

File: Assignment__17/Question3.java
//Write a program to find the maximum of two numbers.

import java.util.Scanner;

class logic
{
    int DisplayMaximum(int iNo1, int iNo2)
    {
        if(iNo1 > iNo2)
        {
            return iNo1;
        }
        else
        {
            return iNo2;
        }
    }
}

class Question3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the first number : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter the second number : ");
        int iValue2 = sobj.nextInt();

        int iResult = 0;

        logic lobj = new logic();
        iResult = lobj.DisplayMaximum(iValue1, iValue2);

        System.out.println("The maximum number is " + iResult);
        
    }    


} 
